### PR TITLE
Adding commitment and live benefit details

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

**Comment by lmichael107** [comment]
If @osg-cat can add me as a contributor (or some other level), I can assign reviewers.

**Comment by osg-cat** [comment]
Ok, try now.

File: docs/index.md [hunk 1]
```diff
@@ -2,32 +2,43 @@
 
 <p style="font-size: larger; font-weight: bold;">Transform your research with vast amounts of computing!</p>
 
-The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC) systems&nbsp;&mdash;
-locally or using the OSG&nbsp;&mdash;
+The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC)&nbsp;&mdash;
```
Comment (osg-cat) [inline docs/index.md:5]: I’m mostly fine with this change, except (a) I would omit the scare quotes (that is an overused and under-meaningful typographical tactic, IMO), and (b) we should stick to simply “OSG” as Miron has requested.

File: docs/index.md [hunk 2]
```diff
@@ -2,32 +2,43 @@
...
+whether on a 'local' system or using the 
+[Open Science Grid (OSG)](https://www.youtube.com/watch?v=t2PzIy-vvaE&list=PLBWb4iScSWcPy7LQ4BuXmm8Z2xO4ZW1J_)&nbsp;&mdash;
 to run large-scale research simulation, analysis, and other applications for cutting-edge science.
-Through lecture, demonstration, hands-on exercises, and one-on-one consulting with experienced OSG staff,
+Through lecture, demonstration, hands-on exercises, and peronalized consulting with experienced OSG staff,
```
Comment (osg-cat) [inline docs/index.md]: Typo, and what’s the benefit here?

File: docs/index.md [hunk 3]
```diff
@@ -2,32 +2,43 @@
...
-The School is ideal for graduate students in any research domain
-where large-scale computing is a vital part of the research process,
-plus we will consider applications from advanced undergraduates, post-doctoral students, faculty, and staff.
-While no travel is involved and the schedule allows for some flexibility,
-the expectation is that participants commit to about 20 hours each week.
-
-**Applications will open in late April or early May;** check back here then.
-
-Note: We hope to return to an in-person OSG User School in 2022!
+The School is ideal for graduate students in any research area for whom beyond-the-desktop computing
```
Comment (osg-cat) [inline docs/index.md]: Is this meant to start a new paragraph? Also, I think the term “beyond-the-desktop computing” has meaning to you, but not the world in general; and it is likely to be a more challenging neologism for non-native speakers. I aim to keep things simple…

File: docs/index.md [hunk 4]
```diff
@@ -2,32 +2,43 @@
...
-where large-scale computing is a vital part of the research process,
-plus we will consider applications from advanced undergraduates, post-doctoral students, faculty, and staff.
-While no travel is involved and the schedule allows for some flexibility,
-the expectation is that participants commit to about 20 hours each week.
-
-**Applications will open in late April or early May;** check back here then.
-
-Note: We hope to return to an in-person OSG User School in 2022!
+The School is ideal for graduate students in any research area for whom beyond-the-desktop computing 
+is a vital part of the research process; we also accept applications from advanced undergraduates,
```
Comment (osg-cat) [inline docs/index.md]: I still prefer “consider” over “accept” here.

File: docs/index.md [hunk 5]
```diff
@@ -2,32 +2,43 @@
...
-the expectation is that participants commit to about 20 hours each week.
-
-**Applications will open in late April or early May;** check back here then.
-
-Note: We hope to return to an in-person OSG User School in 2022!
+The School is ideal for graduate students in any research area for whom beyond-the-desktop computing 
+is a vital part of the research process; we also accept applications from advanced undergraduates, 
+post-doctoral students, faculty, and staff!
+
+_We hope to return to an in-person OSG User School in 2022!_ This virtual offering has been tuned for remote
```
Comment (osg-cat) [inline docs/index.md]: Previously, I thought a lot about the placement of the “we hope to return” part, and I much prefer it further down. This page is about 2021. While I understand some people may care about our plans, I would prefer it to be de-emphasized.

File: docs/index.md [hunk 6]
```diff
@@ -2,32 +2,43 @@
...
+The School is ideal for graduate students in any research area for whom beyond-the-desktop computing 
+is a vital part of the research process; we also accept applications from advanced undergraduates, 
+post-doctoral students, faculty, and staff!
+
+_We hope to return to an in-person OSG User School in 2022!_ This virtual offering has been tuned for remote 
+participation, with the goal of helping attendees get their own work running within a two-week period, and 
+is not merely a translation of the in-person program (last offered in [2019](https://opensciencegrid.org/user-school-2019/)).
+
+**The Virtual School will be held the weeks of August 2&ndash;6 and 9&ndash;13**. 
+Participation is **free** and with materials publicly posted. A selective application process is required
```
Comment (osg-cat) [inline docs/index.md]: How about “Participation is free and materials will be posted publicly.”?

File: docs/index.md [hunk 7]
```diff
@@ -2,32 +2,43 @@
...
+post-doctoral students, faculty, and staff!
+
+_We hope to return to an in-person OSG User School in 2022!_ This virtual offering has been tuned for remote 
+participation, with the goal of helping attendees get their own work running within a two-week period, and 
+is not merely a translation of the in-person program (last offered in [2019](https://opensciencegrid.org/user-school-2019/)).
+
+**The Virtual School will be held the weeks of August 2&ndash;6 and 9&ndash;13**. 
+Participation is **free** and with materials publicly posted. A selective application process is required 
+for live participation, which includes access to HTC systems via the 
+[Center for High Throughput Computing](https://chtc.cs.wisc.edu/approach.shtml) (at UW-Madison)
```
Comment (osg-cat) [inline docs/index.md]: First mention, so “University of Wisconsin&ndash;Madison”. Or more simply, omit the parenthetical, since that information can be determined via the link and is not critical to the message.

File: docs/index.md [hunk 8]
```diff
@@ -2,32 +2,43 @@
...
+and [OSG Connect](https://www.osgconnect.net/) service (otherwise available to researchers working for 
+on US-based academic/govt/non-profit research project).
```
Comment (osg-cat) [inline docs/index.md]: “on a U.S.-based academic, government, or non-profit research project”

File: docs/index.md [hunk 9]
```diff
@@ -2,32 +2,43 @@
...
+
+While **no travel is involved and the schedule allows for some flexibility**,
+applicants are asked to indicate a commitment of about 20 hours each week. You can view [the high-level schedule](schedule.md),
```
Comment (osg-cat) [inline docs/index.md]: Hard to parse, IMO. Simpler? “applicants are asked if they could commit to 20 hours each week.”

File: docs/index.md [hunk 10]
```diff
@@ -2,32 +2,43 @@
...
+[Center for High Throughput Computing](https://chtc.cs.wisc.edu/approach.shtml) (at UW-Madison)
+and [OSG Connect](https://www.osgconnect.net/) service (otherwise available to researchers working for 
+on US-based academic/govt/non-profit research project).
+
+While **no travel is involved and the schedule allows for some flexibility**,
+applicants are asked to indicate a commitment of about 20 hours each week. You can view [the high-level schedule](schedule.md), 
+as well as the detailed schedule and materials from the 
+[2020 OSG Virtual School Pilot](https://opensciencegrid.org/virtual-school-pilot-2020/) 
+This year's schedule may differ slightly, but materials will also be posted within weeks of the live event. 
+Thus, applicants are asked to discuss how the
```
Comment (osg-cat) [inline docs/index.md]: I would omit this sentence — it is relevant iff one applies.

File: docs/index.md (hunk 1 above)
Comment (lmichael107) [reply docs/index.md:5]: fine by me

File: docs/index.md (hunk 2 above)
Comment (lmichael107) [reply docs/index.md]: I like emphasizing 'personalized' (thanks for the typo catch) and this gives us room to meet with more than one person at a time, if we need/want to.
